### PR TITLE
Flavio api place ship -> develop

### PR DESCRIPTION
**Pull request by LoggiExp**

### CODE REVIEW

File: Controllers/PlayersController.cs [hunk 1]
```diff
@@ -68,6 +69,29 @@ public async Task<ActionResult<Player>> PostPlayer(List<NewGamePlayer> newGamePl
             return Ok("Successfully added");
         }
 
+        [HttpPost("placeShips")]
+        public async Task<ActionResult<Ship>> PlaceShip(PlayerShips playerShips)
+        {
+            var playerGrid = await _context.Grids.Where(grid => grid.PlayerId == playerShips.PlayerId).FirstAsync();
+            var playerGridCells = await _context.Cells.Where(cell => cell.GridId == playerGrid.Id).ToListAsync();
+            var ships = await _context.Ships.Where(ship => ship.PlayerId == playerShips.PlayerId).ToListAsync();
```
Comment (RedGuardiaN7) [inline Controllers/PlayersController.cs]: Change var into specific type (Grid, List<Cell>, Ship)

File: Controllers/PlayersController.cs [hunk 2]
```diff
@@ -1,6 +1,7 @@
 ﻿using Microsoft.AspNetCore.Mvc;
 using Microsoft.EntityFrameworkCore;
 using Battleship_APIs.Models;
+using Microsoft.CodeAnalysis.Elfie.Diagnostics;
```
Comment (RedGuardiaN7) [inline Controllers/PlayersController.cs]: A che cosa serve?

**Review by RedGuardiaN7 — COMMENTED** [review]
Request

File: Controllers/PlayersController.cs (hunk 2 above)
Comment (LoggiExp) [reply Controllers/PlayersController.cs]: done

File: Controllers/PlayersController.cs (hunk 1 above)
Comment (LoggiExp) [reply Controllers/PlayersController.cs]: done

**Pull request merged**